03-Bash/find_dup_files/find_dup_files.sh

```diff
@@ -0,0 +1,33 @@
+! #bin/bash
+
+if [[ $1 = "-h" || $1 = "--help" ]]; then
+    HELP_FLAG=1
+    echo "Program prints file names and file size for files with same md5 checksum"
+    exit 0
+else
+    if [[ $1 = "" ]]; then
+        DIR=$(pwd)
+    else     
+        DIR=$1
+    fi
+HELP_FLAG=0
+fi
+
+if [[ -d "$DIR" ]]; then
+    touch -c temp_hash_file.md5
+    :> temp_hash_file.md5
+    LIST_FILES=$(find $DIR -maxdepth 1 -type f)
+        for item in $LIST_FILES; do
+            md5sum "$item" >> temp_hash_file.md5
+        done
```

Comment:
Is it possible to use one line without temporary variable LIST_FILES instead of 4 lines?

Reply:
Next step is to check possibility to make pipe instead of temporary file.